config/examples/cors.yml

```diff
@@ -0,0 +1,24 @@
+cors: &default
+  enabled: false
+  allow:
+  - origins: '*'
+    resources:
+    - !ruby/regexp /\.(?:woff2?|otf|ttf|svg|eot)$/
+    headers: 'x-requested-with'
+    methods: :get
+    max_age: 3628800
+    credentials: false
```

Comment:
This is something I changed from initial implementation. This is an array of origin/resources/other_options bundles. Reading on the web it seems like sometimes multiple allow configurations are desirable. If you think that only a single origin/resources/other_options is well enough, then I can revert.

Actually rack-cors allows in a single `allow` rule to have multiple resources with different other_options (headers, methods, vary, etc.) but I thought this is too fine grained.

The example `cors.yml` replicates what we had with `font_assets`.